src/lib/storage/frame_of_reference_column.cpp

```diff
@@ -8,12 +8,12 @@
 namespace opossum {
 
 template <typename T, typename U>
-FrameOfReferenceColumn<T, U>::FrameOfReferenceColumn(const pmr_vector<T> block_minima,
-                                                     const pmr_vector<bool> null_values,
+FrameOfReferenceColumn<T, U>::FrameOfReferenceColumn(const pmr_vector<T>& block_minima,
```

Comment:
I guess Max passed these by value intentionally, so that data doesn't need to be copied after encoding?

Reply:
Not sure how that makes any difference. You either copy it when you call the method and move it into the member our you pass it in by reference and copy it into the member.

Reply:
Well, since `block_minima` is const, `std::move` doesn't do anything. Either we revert to the old version and remove the const, or we change to &&.

Reply:
That is also why clang-tidy complained. It said that the move didn't do anything.

Reply:
I mean, what?! A std::move can not work? f...

anyway, then the column constructors should be pass-by-non-const-value

Reply:
https://stackoverflow.com/questions/28595117/why-can-we-use-stdmove-on-a-const-object